src/Microsoft.ML/LearningPipeline.cs

```diff
@@ -154,23 +154,29 @@ public LearningPipeline()
                     step = currentItem.ApplyStep(step, experiment);
                     if (step is ILearningPipelineDataStep dataStep && dataStep.Model != null)
                         transformModels.Add(dataStep.Model);
-                     
+
                     else if (step is ILearningPipelinePredictorStep predictorDataStep)
```

Comment:
Nit: I think this change here was unintentional. :D

Reply:
I just ctrl+k+d this file, and this is what VS produce to me. It just replace line full of spaces to empty line.

Reply:
I'd rather you hadn't inserted two completely empty lines here *at all*.

---
In reply to: [188751411](https://github.com/dotnet/machinelearning/pull/172#discussion_r188751411) [](ancestors = 188751411)

Reply:
FALSE ALLEGATIONS! It's actually specific of codeflow. If you look on github PR or merge code, you will see it has only one line, and first line actually suppose to be removed, but for some reason codeflow mark it as yellow instead of red (addition instead of deletion.

---
In reply to: [188834888](https://github.com/dotnet/machinelearning/pull/172#discussion_r188834888) [](ancestors = 188834888,188751411)